test/Cnblogs.Architecture.IntegrationTests/DaprTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Net;
+using FluentAssertions;
+
+namespace Cnblogs.Architecture.IntegrationTests;
+
+[Collection(DddWebTestCollection.Name)]
+public class DaprTests
+{
+    private readonly HttpClient _httpClient;
+
+    public DaprTests(DddWebTestFactory factory)
+    {
+        _httpClient = factory.CreateClient();
+    }
+
+    [Fact]
+    public async Task Dapr_subscribe_endpoint_is_ok()
+    {
+        var response = await _httpClient.GetAsync("/dapr/subscribe");
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+
+        var responseText = await response.Content.ReadAsStringAsync();
+        responseText.Should().Contain("pubsub");
+    }
```

Comment:
```suggestion
    [Fact]
    public async Task Dapr_SubscribeEndpoint_OkAsync()
    {
        // Act
        var response = await _httpClient.GetAsync("/dapr/subscribe");

        // Assert
        response.StatusCode.Should().Be(HttpStatusCode.OK);
        var responseText = await response.Content.ReadAsStringAsync();
        responseText.Should().Contain("pubsub");
    }
```

测试使用的是 AAA 模式（`Arrange-Act-Assert`），测试名称遵循 `测试项目_条件/动作_结果`。